src/Uno.UWP/ApplicationModel/Contacts/ContactPicker.cs

```diff
@@ -0,0 +1,50 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Contacts
+{
+	/// <summary>
+	/// Controls how the Contact Picker user interface opens and what information it shows.
+	/// </summary>
+	public partial class ContactPicker
+	{
+		/// <summary>
+		/// Gets a Boolean value indicating if the contact picker is supported on the current platform.
+		/// </summary>
+		/// <returns>A Boolean value indicating if the contact picker is supported on the current platform.</returns>
+		/// <remarks>Unsupported platforms return false.</remarks>
+		public static IAsyncOperation<bool> IsSupportedAsync() =>
+			IsSupportedTaskAsync().AsAsyncOperation();
```

Comment:
Even if not used, it would be better if the `IsSupportedTaskAsync` accepts a `CancellationToken`